matdeeplearn/models/alignn.py

```diff
@@ -0,0 +1,410 @@
+from typing import Literal, Optional
+
+import numpy as np
+import torch
+from torch.nn import BatchNorm1d, Linear, Sequential, Sigmoid, SiLU
+from torch.nn import functional as F
+from torch_geometric.data import Data
+from torch_geometric.nn import MessagePassing, global_mean_pool
+from torch_scatter import scatter
+
+from matdeeplearn.common.registry import registry
+from matdeeplearn.models.base_model import BaseModel
+
+
+@registry.register_model("ALIGNN")
+class ALIGNN(BaseModel):
+    def __init__(
+        self,
+        # data: torch_geometric.data.Dataset,
+        alignn_layers: int = 4,
+        gcn_layers: int = 4,
+        atom_input_features: int = 114,
+        edge_input_features: int = 50,
+        triplet_input_features: int = 40,
+        embedding_features: int = 64,
+        hidden_features: int = 256,
+        output_features: int = 1,
+        min_edge_distance: float = 0.0,
+        max_edge_distance: float = 8.0,
+        min_angle: float = 0.0,
+        max_angle: float = torch.acos(torch.zeros(1)).item() * 2,
+        link: Literal["identity", "log", "logit"] = "identity",
+        **kwargs
```

Comment:
Just curious, for this model, are there other arguments that you haven't specified that would make us want to have `**kwargs` here? Or are all of them defined? If they are, let's just remove this to avoid confusion

Reply:
Yes we can remove this; I think this was left over from passing in `data` as an argument, which I believe is now not an issue.